bin/hipcc

```diff
@@ -224,17 +237,17 @@ if ($HIP_PLATFORM eq "clang") {
     if (! defined $HIP_LIB_PATH) {
         $HIP_LIB_PATH = "$HIP_PATH/lib";
     }
-    $HSA_PATH=$ENV{'HSA_PATH'} // "/opt/rocm/hsa";
+    $HSA_PATH=$ENV{'HSA_PATH'} // "$HIP_PATH/../hsa";
```

Comment:
```$ROCM_ROOTDIR/hsa```

Reply:
No need to use ROCM_ROOTDIR as it is an invented variable. HIP_PATH is better, mentioned in HIP readme/docs. Let's stick to the point of removing hardcoded "/opt/rocm" from code.